proxy/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9.16-slim-buster
```

Comment:
Can we setup docker-verify workflow with this Dockerfile?

https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/.github/workflows/docker-verify.yaml

Reply:
OK.  I'll try.  I still have to create the proxy-verify workflow, too.